docs/getting_started.rst

```diff
@@ -394,6 +407,10 @@ and ``couchdb_admin_party`` context helpers.
 
         print('Databases: {0}'.format(client.all_dbs()))
 
+.. warning:: Don't use the ``doc.delete()`` method inside your ``Document``
+             context manager. This method immediately deletes the document on
+             the server.
```

Comment:
Okay, let's advise deleting inside the CM is done via `doc['_deleted'] = True`. That seems to work well.

`doc.delete()` works inside the CM (assuming the doc already exists). Although on exiting the CM we always write a doc too. So using `.delete()` in the CM doesn't really make any sense.

`db.pop('docid')` only deals with the local database object afaict. Noting also that using the `Document` CM doesn't update the local cache.